test/unit/test_assembly.py

```diff
@@ -139,8 +139,7 @@ def test_lassa_protein(self):
         assembly.order_and_orient(
             os.path.join(inDir, 'contigs.lasv.fasta'),
             os.path.join(inDir, 'ref.lasv.fasta'),
-            outFasta,
-            aligner='promer')
+            outFasta)
```

Comment:
Do you mean my change is incorrect, or the test prior to the change? In `master`, `aligner` is commented out from the `order_and_orient` signature as of [this](https://github.com/broadinstitute/viral-ngs/commit/090bf5c9f89f75ae96affc32b514cbbcba5d1bf6) commit. I'll remove the test if we don't need it.